src/OpenTelemetry.Api/Trace/SpanAttribute.cs

```diff
@@ -0,0 +1,121 @@
+﻿// <copyright file="SpanAttribute.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace OpenTelemetry.Trace
+{
+    /// <summary>
+    /// A class that represents the span attributes.
+    /// </summary>
+    public struct SpanAttribute
```

Comment:
`SpanAttributes` instead of `SpanAttribute`? I was just catching up with the conversation you were having with Cijo, so apologies if this was already hashed out.

Reply:
`SpanAttributes` seems more apt.

Reply:
Just updated.